core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -139,28 +145,25 @@ inline bool OpenMP::impl_is_initialized() noexcept {
   return Impl::OpenMPInternal::singleton().is_initialized();
 }
 
-inline bool OpenMP::in_parallel(OpenMP const&) noexcept {
-  // FIXME_OPENMP We are forced to use t_openmp_instance because the function is
-  // static and does not use the OpenMP object
-  return ((Impl::OpenMPInternal::singleton().m_level < omp_get_level()) &&
-          (!Impl::t_openmp_instance ||
-           Impl::t_openmp_instance->m_level < omp_get_level()));
+inline bool OpenMP::in_parallel(OpenMP const& exec_space) noexcept {
+  return (
+      (exec_space.impl_internal_space_instance()->m_level < omp_get_level()) &&
+      (!Impl::t_openmp_instance ||
+       Impl::t_openmp_instance->m_level < omp_get_level()));
```

Comment:
For those who like me struggle to see the diff
`Impl::OpenMPInternal::singleton().m_level -> exec_space.impl_internal_space_instance()->m_level`

Reply:
If you hide whitespace it's pretty obvious what's changed.